userspace/libscap/scap_procs.c

```diff
@@ -260,7 +269,14 @@ int32_t scap_proc_fill_info_from_stats(char* procdirname, struct scap_threadinfo
 	tinfo->pfmajor = pfmajor;
 	tinfo->pfminor = pfminor;
 	tinfo->sid = (uint64_t) sid;
-	tinfo->pgid = (uint64_t) pgid;
+
+	// If we did not find vpgid above, set it to pgid from the
+	// global namespace.
+	if(tinfo->vpgid != 0)
```

Comment:
Is this condition inverted? Seems like it's doing the exact opposite of what the comment says?

Reply:
Yes, that test was backwards, fixed. Thanks for pointing it out.

Reply:
Was this changed again?

Reply:
Oh yes, sorry. I dropped the commit that had this change. Will fix again.